.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+    If this pull request addresses an issue, link to it:
+    "Fixes #xxxx" or "Resolves #xxxx".
+    Also include the issue number in the title "[#xxxx] Title"
+
+    If this pull request is part of an issue, but does not fully fix
+    it, use: "Part of #xxxx".
+-->
+Fixes #xxxx
+
+## Proposed commit message
+<!--
+    Propose a detailed commit message for this pull request using the
+    following format. Wrap lines at 72 characters.
+
+    Guide on how to write a good commit message:
+    https://oss-generic.github.io/process/docs/FormatsAndConventions.html#commit-message
+-->
+```
+Brief description of current behaviour and reason(s) for the change.
+
+Call-to-action statement describing what you are doing in this PR.
+```
+
+## Other information
+<!--
+    Include any other relevant information, such as special testing
```

Comment:
Feels a little blunt. Referencing the `feature-request.md`, maybe phrasing as question would be nicer:

```
Are there other relevant information, such as special testing instructions, 
which will help the reviewer better understand the code?

You may also include a brief description of why the problem occurred.
```

Reply:
Agree, thanks!